azure_pipeline.yaml

```diff
@@ -104,6 +144,19 @@ stages:
           agentPool: ${{ variables.agentPool }}
           terraformVersion: ${{ variables.terraformVersion }}
 
+  - stage: ApplyCftAppsLBSandbox
+    displayName: "Apply CFT APPs LB - Sandbox"
```

Comment:
Could you also add 

`dependsOn: GetArtifacts`

Reply:
its there one line further?